src/handlers.js

```diff
@@ -289,18 +335,25 @@ async function jobHandler(message) {
     }
     // Warn the user know that there was a problem processing their
     // config file with a comment.
-    await context.github.repos.createCommitComment({
+    await instGithub.repos.createCommitComment({
       owner: organization,
       repo: repository,
       sha,
-      body: 'Submitting the task to Taskcluster failed. ' +
-      '\nDetails:\n\n```js\n' +  errorBody + '\n```',
+      body: [
+        '<summary>Submitting the task to TaskCluster failed.',
+        '<details>\n',
+        '```js',
```

Comment:
I switched this around in a commit to master -- The details element should surround the summary element.